pkg/testcase/cluster.go

```diff
@@ -114,3 +114,35 @@ func checkAndPrintAgentNodeIPs(agentNum int, agentIPs []string, isWindows bool)
 		Expect(agentIPs).Should(BeEmpty())
 	}
 }
+
+func ClusterReset() {
+	product, err := shared.Product()
+	Expect(err).NotTo(HaveOccurred())
+
+	g := GinkgoT()
+	cluster := factory.ClusterConfig(g)
+	ip := cluster.ServerIPs[0]
+
+	var cmd string
+	var res string
+	var cmdErr error
+
+	cmd = fmt.Sprintf("sudo %s server --cluster-reset", product)
+
+	switch product {
```

Comment:
Do we need a switch here? I dont see any difference except on k3s it returns a reponse and rke2 it does not and returns an error instead?

Reply:
Yes - k3s prints the information for the cluster reset as a response, whereas for rke2 is it printed as an error.

```
switch product {
	case "k3s":
		res, resetCmdErr = shared.RunCommandOnNode(resetCmd, ip)
		Expect(resetCmdErr).NotTo(HaveOccurred())
		Expect(res).To(ContainSubstring("Managed etcd cluster"))
		Expect(res).To(ContainSubstring("has been reset"))
	case "rke2":
		_, resetCmdErr = shared.RunCommandOnNode(resetCmd, ip)
		Expect(resetCmdErr).To(HaveOccurred())
		Expect(resetCmdErr.Error()).To(ContainSubstring("Managed etcd cluster"))
		Expect(resetCmdErr.Error()).To(ContainSubstring("has been reset"))
	default:
		shared.LogLevel("error", "unsupported product: %s", product)
		g.Fail()
	}
```

Reply:
OK sounds good, not sure if that is by design but it seems a bit weird with inconsistent behavior

Reply:
This was brought up to the team in Slack. We can continue conversations there but currently it's by design, which is definitely weird!